fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergConnector.java

```diff
@@ -30,12 +33,17 @@ public class IcebergConnector implements Connector {
     private static final Logger LOG = LogManager.getLogger(IcebergConnector.class);
 
     private final Map<String, String> properties;
+    private final CloudConfiguration cloudConfiguration;
+    private final HdfsEnvironment hdfsEnvironment;
     private final String catalogName;
     private ConnectorMetadata metadata;
 
     public IcebergConnector(ConnectorContext context) {
         this.catalogName = context.getCatalogName();
         this.properties = context.getProperties();
+        this.cloudConfiguration = CloudConfigurationFactory.buildStorageCloudConfiguration(properties);
+        this.hdfsEnvironment = new HdfsEnvironment();
+        this.hdfsEnvironment.tryApplyCloudConfiguration(this.cloudConfiguration);
```

Comment:
Why HdfsEnv need to apply cloudConfiguration?  Isn't cloudConfiguration for S3 or OSS？

Reply:
I don't undstand the effect of HdfsEnvironment，it looks like build a new Configuration from cloudConfiguration, How about use one funtion like buildConfiguration(cloudConfiguration) do this?